docs/api/index.html

```diff
@@ -9062,7 +9062,7 @@ <h3 class='fl m0' id='meetings'>
         <div class='space-bottom0'>
           <div>
             <span class='code bold'>destination</span> <code class='quiet'>(<a href="https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String">string</a>)</code>
-	    sipURL, spaceId, phonenumber, or locus object}
+	    sipURL, phonenumber, or locus object}
```

Comment:
Why do we have to check-in this file? 

Shouldn't this be extracted from the code rather than a manual update?

Reply:
@mkesavan13, Correct, ignoring this file not checking-in this file now.